### PR TITLE
Swift 3.0, Updated for Xcode 8 beta 4

### PR DESCRIPTION
**Pull request by jakerockland**

### CODE REVIEW

**Comment by hartbit** [comment]
Looks good to me 👍

File: Source/Supporting Files/Utilities.swift [hunk 1]
```diff
@@ -117,7 +117,7 @@ extension DispatchTimeInterval {
 
 extension DispatchQueue {
```
Comment (jakerockland) [inline Source/Supporting Files/Utilities.swift]: @hartbit Does this part look good to you as well, this is the only place I was a bit uncertain.

Comment (delba) [inline Source/Supporting Files/Utilities.swift]: Can't we just remove the extension?

Comment (jakerockland) [inline Source/Supporting Files/Utilities.swift]: I believe so. 😅 Why was it originally there?

Comment (delba) [inline Source/Supporting Files/Utilities.swift]: `DispatchQueue` hasn't always been that nice ;)

**Comment by delba** [comment]
Thank you @jakerockland :)

**Comment by jakerockland** [comment]
No prob! 😄

**Pull request merged**